src/lib/atoms/buttons/button.svelte

```diff
@@ -5,10 +5,8 @@
 </script>
 
 <button style="background-color: {buttonBackground}; color: {buttonColor};">
```

Comment:
Om inhoud en vormgeving beter te scheiden, zou je hier beter gebruik kunnen maken van dynamische custom properties, en dus niet het direct stylen van de betreffende properties in HTML.

Bekijk hier een voorbeeld van wat ik bedoel:

https://github.com/fdnd-agency/visual-thinking/blob/44729eb227f27b1ce2c5404e0b571d4c696f2fdb/src/lib/atoms/breadcrumb.svelte#L1-L7